config/owasp/suppressions.xml

```diff
@@ -31,16 +34,17 @@
     <cve>CVE-2016-1000027</cve>
```

Comment:
Remove this line as it's now in the false positives section.  There were two instances of it here and one was removed previously.

Reply:
Duplicate has been removed.